test/gtest/ucs/test_config.cc

```diff
@@ -246,6 +253,21 @@ class test_config : public ucs::test {
         return UCS_LOG_FUNC_RC_CONTINUE;
     }
 
+    static ucs_log_func_rc_t
+    config_error_suppress(const char *file, unsigned line, const char *function,
```

Comment:
can reuse wrap_errors_logger ?

Reply:
technically yes, but in this case we can't check if error is handled - there is no errors counter in ```wrap_errors_logger```

Reply:
can we call wrap_errors_logger() after counting, to print err message?

Reply:
ok, wrapped